1-js/02-first-steps/09-comparison/article.md

```diff
@@ -1,214 +1,191 @@
-# Comparisons
+# तुलना
 
-We know many comparison operators from maths.
+हम गणित से कई तुलना ऑपरेटरों को जानते हैं।
 
-In JavaScript they are written like this:
+जावास्क्रिप्ट में वे इस तरह लिखे जाते हैं:
 
-- Greater/less than: <code>a &gt; b</code>, <code>a &lt; b</code>.
-- Greater/less than or equals: <code>a &gt;= b</code>, <code>a &lt;= b</code>.
-- Equals: `a == b`, please note the double equality sign `==` means the equality test, while a single one `a = b` means an assignment.
-- Not equals. In maths the notation is <code>&ne;</code>, but in JavaScript it's written as <code>a != b</code>.
+- ग्रेटर / से कम: <code>a &gt; b</code>, <code>a &lt; b</code>।
+- ग्रेटर / से कम या बराबर: <code>a &gt;= b</code>, <code>a &lt;= b</code>।
+- समतुल्य: `a == b`, कृपया दोहरे समानता चिन्ह पर ध्यान दें `==` का अर्थ है समानता परीक्षण, जबकि एकल `a = b` का अर्थ है असाइनमेंट।
+- नहीं के बराबर है। गणित में अंकन<code>&ne;</code>है, लेकिन जावास्क्रिप्ट में इसे<code>a != b</code> लिखा जाता है।
 
-In this article we'll learn more about different types of comparisons, how JavaScript makes them, including important peculiarities. 
+इस लेख में हम विभिन्न प्रकार की तुलनाओं के बारे में अधिक जानेंगे, कि कैसे जावास्क्रिप्ट उन्हें महत्वपूर्ण विशिष्टता सहित बनाता है।
+अंत में आपको "जावास्क्रिप्ट क्विर्क" से संबंधित मुद्दों से बचने के लिए एक अच्छा नुस्खा मिलेगा।
 
-At the end you'll find a good recipe to avoid "javascript quirks"-related issues.
+# बुलियन परिणाम है
+सभी तुलना ऑपरेटर एक बूलियन मान लौटाते हैं:
 
-## Boolean is the result
+- सत्य - का अर्थ है "हाँ", "सही" या "सत्य"।
```

Comment:
true, false are javascript keywords hence we shouldn't be translating them.

Reply:
/done